docs/_includes/include-lines-tags.adoc

```diff
@@ -32,9 +32,10 @@ include::ex-include.adoc[tags=tag-co]
 IMPORTANT: The `tag::[]` and `end::[]` directives should be placed after a line comment as defined by the language of the source file.
 The directives must also appear at the _end_ of the line.
 In the previous example, we choose to prefix the lines with a hash (`#`) because that's the start of a line comment in Ruby.
+For XML files you can use the `+<!-- tag::name[] -->+` and `+<!-- end::name[] -->+` delimiters.
 
-// moved from include-directive.adoc, since I assume it refers to tagging. Should "targets" be tags?
-TIP: Asciidoctor supports targets that include spaces and the `\{sp}` attribute references.
+// moved from include-directive.adoc, since I assume it refers to tagging.
+TIP: Asciidoctor supports tags that include spaces and the `\{sp}` attribute references.
```

Comment:
"target" is the right term here. We are referring to the target of the include directive, otherwise known as the path being included. Spaces are permitted in the filename. As an alternative, the `{sp}` attribute reference is also recognized (though no longer preferred).